src/main/java/org/primefaces/component/datatable/DataTable.java

```diff
@@ -294,18 +297,6 @@ public void processValidators(FacesContext context) {
         }
     }
 
-    @Override
```

Comment:
@tandraschko @melloware I remember you guys faced an issue with this code recently, but I can't recall what it was... Can you remind me what was the issue number?